tests/phpunit/elementor/includes/test-frontend.php

```diff
@@ -118,4 +137,35 @@ public function test_get_early_access_google_font_urls() {
 		// Assert
 		$this->assertContains( 'https://fonts.googleapis.com/earlyaccess/opensanshebrew.css', $font_urls );
 	}
+
+	public function test_widget_with_global_scripts() {
+		// Arrange
+		$frontend = new Frontend();
+		$frontend->register_scripts();
+
+		$widget_with_global_scripts = $this->elementor()->widgets_manager->get_widget_types( 'text-editor' );
```

Comment:
Maybe mention with comment or with the variable name, that this widget should be V3 widget and it does not matter which widget it should be. also for the V4 widget